db/migrate/20230606190606_add_year_to_teams.rb

```diff
@@ -0,0 +1,15 @@
+class AddYearToTeams < ActiveRecord::Migration[7.0]
+  def up
+    add_column :teams, :year, :integer
+    
+    Team.find_each do |team|
+      puts "Updating Team with id: #{team.id}"
+      team.update! year: 2022
+      puts "Successfully updated Team with id: #{team.id}"
+    end
```

Comment:
You can update every record in the db in a single SQL statement with `Team.update_all(year: 2022)` instead. It'll be much faster than doing each one individually.

Reply:
Yeah this is true. The one reason I told him to do it like this was to talk about printing the messages about what was updating and the success state. Plus we only had ~80 records so I wasn't concerned about the speed.

Reply:
I'd still put the logging messages around the update_all so you know what's going on.

Also these probably should be separate up and down methods if we're going to modify the existing records like that?

Reply:
Wow, that is fassstttttt.